File: QE_FST_Selenium_python/Activity9_keyboardEvents.py
from selenium import webdriver
import time
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/keyboard-events")
    keyboard=driver.find_element(By.XPATH,"/html/body/div/main/div/div/div/div/div[2]/h1")
    time.sleep(3)
    print(keyboard.text)

File: QE_FST_Selenium_java/Activity9_keyboardEvents.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity9_keyboardEvents {
    public static void main(String args[]) throws InterruptedException
    {
        WebDriver driver=new FirefoxDriver();
        driver.get("https://training-support.net/webelements/keyboard-events");
        System.out.println(driver.getTitle());
        WebElement keyboard=driver.findElement(By.xpath("/html/body/div/main/div/div/div/div/div[2]/h1"));
        Thread.sleep(3000);
        //keyboard.sendKeys("Hello");
        System.out.println(keyboard.getText());
        driver.quit();

    }
}
